src/lib/output/ResultsProcessorFactory.ts

```diff
@@ -0,0 +1,35 @@
+import {Display} from "../Display";
+import {JsonReturnValueHolder} from "./JsonReturnValueHolder";
+import {RunOutputOptions, RunResultsProcessor} from "./RunResultsProcessor";
+import {CompositeResultsProcessor, ResultsProcessor} from "./ResultsProcessor";
+import {ENV_VAR_NAMES} from "../../Constants";
+import {inferFormatFromInternalOutfile, OutputFormat} from "./OutputFormat";
```

Comment:
Personal preference: I prefer to import methods via namespace imports (`import * as OutputFormat` or similar), because it means the method gets referenced in code as `OutputFormat.whatever()`, which keeps it obvious that the method is being imported from somewhere else instead of declared locally.

Reply:
This makes sense but I wouldn't want OutputFormat.OutputFormat. How would you avoid that?

Reply:
If you make the OutputFormat enum a `default` export, then you could do:
```
import OutputFormat, * as OutputFormatNS from './OutputFormat';
```
Then you could reference OutputFormat as itself, and the method as `OutputFormatNS.blah()`.